packages/vats/src/bootstrap.js

```diff
@@ -68,15 +65,13 @@ export function buildRootObject(vatPowers, vatParameters) {
       board,
       chainTimerService,
       zoe,
-      contractHost,
       { priceAuthority, adminFacet: priceAuthorityAdmin },
     ] = await Promise.all([
       E(vats.sharing).getSharingService(),
       E(vats.registrar).getSharedRegistrar(),
       E(vats.board).getBoard(),
       E(vats.timer).createTimerService(timerDevice),
       /** @type {ERef<ZoeService>} */ (E(vats.zoe).buildZoe(vatAdminSvc)),
-      E(vats.spawner).buildSpawner(vatAdminSvc),
```

Comment:
yay deleting the on-chain spawner!

Reply:
Yeah, I saw this and went: "oh boy, that looks like a backdoor even if Zoe is locked down."